src/installer/pkg/projects/Microsoft.NETCore.DotNetHost/runtime.osx.Microsoft.NETCore.DotNetHost.props

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Delete the file with upper-cased name?

Reply:
I double checked the upper case files are not present, for some reason these files are not being shown as rename by github

Reply:
I guess you are on Windows and git got confused by the Windows case-insensitive file system.

You need to go to Unix where you can see the files with different casing and clean it up there: 
```
jkotas@JKOTAS5:~/runtime/src/installer/pkg/projects/Microsoft.NETCore.DotNetHost$ ls -la
total 36
drwxrwxrwx 1 jkotas jkotas 4096 Mar  9 16:29 .
drwxrwxrwx 1 jkotas jkotas 4096 Mar  9 16:29 ..
-rw-rw-rw- 1 jkotas jkotas 2654 Mar  9 16:24 host.wxs
-rw-rw-rw- 1 jkotas jkotas 2039 Mar  9 16:24 Microsoft.NETCore.DotNetHost.pkgproj
-rw-rw-rw- 1 jkotas jkotas  439 Mar  9 16:29 runtime.freebsd.Microsoft.NETCore.DotNetHost.props
-rw-rw-rw- 1 jkotas jkotas  439 Mar  9 16:24 runtime.FreeBSD.Microsoft.NETCore.DotNetHost.props
-rw-rw-rw- 1 jkotas jkotas  439 Mar  9 16:29 runtime.osx.Microsoft.NETCore.DotNetHost.props
-rw-rw-rw- 1 jkotas jkotas  439 Mar  9 16:24 runtime.OSX.Microsoft.NETCore.DotNetHost.props
-rw-rw-rw- 1 jkotas jkotas  439 Mar  9 16:29 runtime.unix.Microsoft.NETCore.DotNetHost.props
-rw-rw-rw- 1 jkotas jkotas  439 Mar  9 16:24 runtime.Unix.Microsoft.NETCore.DotNetHost.props
-rw-rw-rw- 1 jkotas jkotas  443 Mar  9 16:29 runtime.win.Microsoft.NETCore.DotNetHost.props
```

Reply:
okay i will do that.